react/features/base/testing/components/TestConnectionInfo.js

```diff
@@ -204,6 +215,7 @@ class TestConnectionInfo extends Component<Props, State> {
  * @returns {{
  *     _conferenceConnectionState: string,
  *     _conferenceJoinedState: string,
+ *     _grantModeratorEnabled: string,
```

Comment:
Remove all of there and say {Props} please.

Reply:
done!